include/libdnf/plugin/iplugin.hpp

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright (C) 2021 Red Hat, Inc.
+
+This file is part of microdnf: https://github.com/rpm-software-management/libdnf/
+
+Microdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
```

Comment:
Did you mean this to be under the GPL?  The COPYING file claims libdnf is LGPLv2+.

Reply:
No. I overlooked. Incorrectly used the license header for microdnf instead of libdnf.